LibOS/glibc-2.19.patch

```diff
@@ -664,41 +664,10 @@ index 6dcbabc..c87c773 100644
    LIBC_PROBE (init_start, 2, LM_ID_BASE, r);
  
    /* Auditing checkpoint: we are ready to signal that the initial map
-@@ -1605,140 +1654,11 @@ ERROR: ld.so: object '%s' cannot be loaded as audit interface: %s; ignored.\n",
```

Comment:
This patch used to disable LD_PRELOAD in glibc. There is no strong reason that we should do this, and having LD_PRELOAD can be helpful in many cases.